extensions/blocks/social-previews/modal.scss

```diff
@@ -0,0 +1,158 @@
+/**
+ * Modal styles for Social Previews
+ */
+
+@import '../../shared/styles/gutenberg-base-styles.scss';
+
+// Styles shared between modals
+.jetpack-social-previews__modal {
+	.components-modal__header {
+		margin: 0;
+	}
+
+	.components-modal__content {
+		padding: 0;
+	}
+}
+
+// Main modal containing the social previews
+.jetpack-social-previews__modal-previews {
+	display: flex;
+	flex-direction: column;
+	height: 100%;
```

Comment:
This line doesn't seem to change anything for me when I remove it. Is it necessary?

Reply:
oooh I see the mobile version now. Nevermind.

Reply:
I didn't write the CSS here but from what I can tell it's needed for the container to cover the full height when viewed on mobile. Given the mobile first approach it makes sense for this to be in the default declaration of that class (and not a mobile specific media query).